### PR TITLE
Login

### PR DESCRIPTION
**Pull request by MuradSh**
- Implemented Sign Up and Sign In designs with @oankit 
- @oankit  wrote tests for SignUp

### CODE REVIEW

File: code/signup.css
Comment (ammaarkhan) [inline code/signup.css:1]: can probably remove this file to reduce confusion.

File: code/signup.html
Comment (ammaarkhan) [inline code/signup.html:1]: can probably remove this file to reduce confusion.

**Review by ammaarkhan — CHANGES_REQUESTED** [review]
can probably remove files directly under path: code/ to reduce confusion.

**Review by ribhavsharma — APPROVED** [review]
LGTM! 💯

**Pull request merged**